core/src/main/kotlin/entity/Icon.kt

```diff
@@ -43,16 +46,11 @@ public sealed class Icon(
     public class EmojiIcon(animated: Boolean, emojiId: Snowflake, kord: Kord) :
         Icon(if (animated) Image.Format.GIF else Image.Format.WEBP, animated, DiscordCdn.emoji(emojiId), kord)
 
-    public class DefaultUserAvatar(discriminator: Int, kord: Kord) :
```

Comment:
again, could you restore the formatting here? this will make it easier to review this pr